src/client/src/App.tsx

```diff
@@ -37,19 +37,9 @@ export async function initApp() {
     </StrictMode>,
   )
 
-  const { ga } = window
-
-  ga("create", {
-    trackingId: GA_TRACKING_ID,
-    transport: "beacon",
-  })
-
-  ga("set", {
-    dimension1: gaDimension,
-    dimension2: gaDimension,
-    dimension3: import.meta.env,
-    page: window.location.pathname,
-  })
-
-  ga("send", "pageview")
+  window.gtag("js", new Date())
+  window.gtag(
+    "config",
+    ENVIRONMENT === "prod" ? "G-Z3PC9MRCH9" : "G-Y28QSEPEC8",
```

Comment:
Should these value come from `.env` file?

Reply:
hmm .. I was about to say yes, but then I thought about running a Prod build locally .. this ofc does not save you from that case either .. I'll think again on this, we kind of need a dynamic check .. but that is going to be difficult as it is invoked in the `index.html` .. 🤔